src/Content/Changes.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace Kirby\Content;
+
+use Kirby\Cms\App;
+use Kirby\Cms\Files;
+use Kirby\Cms\ModelWithContent;
+use Kirby\Cms\Pages;
+use Kirby\Cms\Site;
+use Kirby\Cms\Users;
+
+/**
+ * The Changes class tracks changed models
+ * in the Site's changes field.
+ *
+ * @package   Kirby Content
+ * @author    Bastian Allgeier <bastian@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://getkirby.com/license
+ */
+class Changes
+{
+	/**
+	 * Access helper for the field, in which changes are stored
+	 */
+	public function field(): Field
+	{
+		return $this
+			->site()
+			->version(VersionId::published())
+			->content()
+			->get('changes');
```

Comment:
`Suggestion:` To avoid more protected filenames, how about using `_changes` or `__changes`?

Reply:
I'm really torn with the underscore fieldnames. Same for the _lock field. The `Site::changes()` method is already kind of protecting the field by avoiding direct access. On the other hand, it's definitely more likely that a changes field is already used. It also feels a bit like we should have made that decision in other places too and used it for sort or template or other such system fields. I think I'm fine with both.